pkgs/development/python-modules/roundrobin/default.nix

```diff
@@ -0,0 +1,23 @@
+{ buildPythonPackage
+, fetchFromGitHub
+, lib
+}:
+
+buildPythonPackage rec {
+  pname = "roundrobin";
+  version = "0.0.2";
+
+  src = fetchFromGitHub {
+    owner = "linnik";
+    repo = pname;
+    rev = "5a5b34f2cd3ab8aa6d4135b31ab512b2d9338c19";
+    sha256 = "sha256-h+GcsHM0D4uBMcOng6mioC5o5AWbegiGqGYtuPoduM4=";
+  };
+
```

Comment:
Can you add an additional pythonImportsCheco here?

Reply:
sure, good idea. i've added that now.